release_notes.md

```diff
@@ -1,21 +1,10 @@
 # What's New
 
-## Introducing Top SOC Questions
-
-Edit or disable existing prompts or add new prompts based on your security posture.
-
-- You can now customize your interactions with the LLM by providing specific details and guidelines for your chats. Define your SOC questions to customize your LLM interactions.
-
-- Specific modules now have pre-defined questions. For example, under the **Alerts** module, questions include:
-
-    - Get Alert Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
-    
-    Similarly, for the **Incident** module, questions may be:
-    
-    - Get Incident Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
+- The AI Assistant can understand and respond to context sensitive conversations and perform given SOC tasks
```

Comment:
bot is also cable to to ask you questions if it needs any information/contxt to perform the task like Bloc IOC prompt, bot will ask you IOC value, type, reason, etc.